treccovid_semantic_search/workload.json

```diff
@@ -0,0 +1,32 @@
+{% import "benchmark.helpers" as benchmark with context %}
+
+{
+  "version": 2,
+    "description": "Trec-Covid is a dataset collection of documents about COVID-19 information.",
+  "indices": [
+    {
+      "name": "treccovid",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "treccovid",
+      "base-url": "https://vesa-oswl.s3.us-west-2.amazonaws.com/treccovid",
```

Comment:
Who is maintaining this service?

Reply:
Its my aws account. I can copy the corpus to the opensearch official cloud, if I can get instruction how to do it.

Reply:
one option for hosting the dataset is to add it as a release artifact to one of opensearch repos (existing or new). This way it will be hosted at github site and will be available to public with direct link